test/server/server_fuzz_test.cc

```diff
@@ -52,6 +52,10 @@ makeHermeticPathsAndPorts(Fuzz::PerTestEnvironment& test_env,
     }
   }
   for (auto& cluster : *output.mutable_static_resources()->mutable_clusters()) {
+    for (auto& health_check : *cluster.mutable_health_checks()) {
+      // QUIC is not enabled in production code yet, so remove references for HTTP3.
```

Comment:
Should we have an issue created and referenced here to track re-enabling?

Reply:
All set (and referenced)